packages/webpack-config/webpack/loaders/createBabelLoaderAsync.js

```diff
@@ -96,18 +97,23 @@ module.exports = async function({
     },
     use: {
       ...customUse,
-      // AFAIK there is no reason to replace `babel-loader`.
-      loader: require.resolve('babel-loader'),
-
+      loader: path.join(__dirname, 'expo-babel-loader'),
       options: {
         // TODO: Bacon: Caching seems to break babel
         cacheDirectory: false,
         // Explicitly use babel.config.js instead of .babelrc
         babelrc: false,
         // Attempt to use local babel.config.js file for compiling project.
         configFile: true,
+
         // Only clobber hard coded values.
         ...(customUseOptions || {}),
+
+        custom: {
```

Comment:
Reading the babel-loader source code, it looks like options.caller gets copied into the value used for api.caller -- do we need the custom loader at all?